### PR TITLE
Enable uploading to tracks that not yet comtain an apk.

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

**Comment by lpusok** [comment]
@raquezha working on getting this merged. Meanwhile please upload manually the first version of your app (in every track you need like internal, beta...) in the Play Console, afterwards you can use the step even without this fix.

**Comment by raquezha** [comment]
Thanks for the reply! yes that's the temporary fix for this issue. thanks!

File: main.go [hunk 1]
```diff
@@ -78,17 +78,11 @@ func uploadApplications(configs Configs, service *androidpublisher.Service, appE
 func updateTracks(configs Configs, service *androidpublisher.Service, appEdit *androidpublisher.AppEdit, versionCodes []int64) error {
 	editsTracksService := androidpublisher.NewEditsTracksService(service)
 
-	allTracks, err := getAllTracks(configs.PackageName, service, appEdit)
+	_, err := getAllTracks(configs.PackageName, service, appEdit)
```
Comment (godrei) [inline main.go:81]: why do we need this call? just for logging the available tracks?

Comment (Szabadember) [reply main.go:81]: This call seems unnecessary, the return value is unused and we should not build on side effects which is just logging.

Comment (lpusok) [reply main.go:81]: Yes it was used only for logging, which seemed at least somewhat useful.

Comment (Szabadember) [reply main.go]: ok, let's merge it

**Comment by nirajrajbhandari** [comment]
👏

**Comment by nirajrajbhandari** [comment]
@Szabadember  looks like `3.1.0` is not yet available on bitrise collection.

Screen Shot 2020-07-15 at 7 21 37 PM


when will `3.1.0` be available for use on bitrise?

**Comment by bitce** [comment]
Hi @nirajrajbhandari! This version is not yet released, will be soon with further additions to the step in the upcoming weeks.

**Comment by nirajrajbhandari** [comment]
Thanks @bitce .

**Pull request merged**